src/App.tsx

```diff
@@ -17,18 +60,37 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                filterBy={filterBy}
+                setFilterBy={setFilterBy}
+                query={query}
+                setQuery={setQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {
+                !isLoaded
```

Comment:
```suggestion
                isLoaded
```
та поміняй місцями компоненти навіщо ускладнювати?)